.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Checklist for this pull request
+Before submitting a pull request, please make sure you have to following:
```

Comment:
Haha, yeah it's a bit weird when reading the whole sentence.
Maybe it should be:
```
Before submitting a pull request, please make sure to follow:
```